snapshot/README.md

```diff
@@ -154,6 +154,8 @@ XCUIApplication *app = [[XCUIApplication alloc] init];
 [app launch];
 ```
 
+_Make sure you only have one `launch` call, as Xcode adds one automatically on new test files._
```

Comment:
Maybe let's phrase it like `_Make sure you only have one `launch` call in your test class`